src/bin/regex_capture_date.rs

```diff
@@ -0,0 +1,16 @@
+// captures: Capture returns an Option containing capture groups if the pattern matches the text.
```

Comment:
Is this the recommended way to add multi-line comments at the start of a file in Rust? For example, in C, C++, we would use the /* */ pattern at the start of the file.